orchagent/portsorch.cpp

```diff
@@ -4042,6 +4199,12 @@ bool PortsOrch::removeLagMember(Port &lag, Port &port)
     LagMemberUpdate update = { lag, port, false };
     notify(SUBJECT_TYPE_LAG_MEMBER_CHANGE, static_cast<void *>(&update));
 
+    if (gMySwitchType == "voq")
```

Comment:
Similarly should we check for SwitchId here ? or are we handling it with the parent lag interface ?

Reply:
This lag's switch id is checked in syncing function voqSyncDelLagMember() for whether it is remote lag or not. If the lag is remote, no syncing will happen.